src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1454,7 +1449,7 @@ public void IdleOperations(object state)
                         if (idleDbInstance.CanUnload == false)
```

Comment:
Maybe it would be good to move it to the top? It is easier to read a property than calculate other stuff? I should have done that when I've introduced this.